### ISSUE TITLE
QuotaExceededError when origin quota (supposedly) hasn't been exceeded

### ISSUE THREAD

**Issue by Zloka** [dfahlander/Dexie.js#1355]
My application stores quite a bit of data to retain offline persistence (up to 2GB in extreme cases) using Dexie. The application checks for QuotaExceededErrors and informs the user that they are running out of storage space.

One user is getting my warning continuously. They are using Google Chrome and are reporting that they have used 51,4/118 GB of the available memory on the device.
Per [this article](https://web.dev/storage-for-the-web/), that should still be fine, so I fail to understand why the user is facing this error. Any ideas? 🤔

**Comment by dfahlander** [comment]
Unless anyone else has a good answer, I think this question would be something for the [chromium bug tracker](https://crbugs.com).

**Comment by Zloka** [comment]
Yeah, I see. For what it's worth I posted there as well: https://bugs.chromium.org/p/chromium/issues/detail?id=1231385

Sadly it's a bit hard to get more details out of a single user 🙁
